### ISSUE TITLE
Fix issues raised by style testing for 2.7

### ISSUE THREAD

**Issue by poshul** [OpenMS/OpenMS#5400]
Style testing on develop fails on the following files.

```
The following tests FAILED:
         14 - src/openms/source/ANALYSIS/DECHARGING/FeatureDeconvolution.cpp_cppcheck_test (Failed)
         16 - src/openms/source/ANALYSIS/DECHARGING/MetaboliteFeatureDeconvolution.cpp_cppcheck_test (Failed)
         71 - src/openms/source/ANALYSIS/MAPMATCHING/FeatureGroupingAlgorithmKD.cpp_cppcheck_test (Failed)
        139 - src/openms/source/ANALYSIS/OPENSWATH/TargetedSpectraExtractor.cpp_cppcheck_test (Failed)
        152 - src/openms/source/ANALYSIS/QUANTITATION/IsotopeLabelingMDVs.cpp_cppcheck_test (Failed)
        209 - src/openms/source/CHEMISTRY/ElementDB.cpp_cppcheck_test (Failed)
        228 - src/openms/source/CHEMISTRY/ModificationsDB.cpp_cppcheck_test (Failed)
        324 - src/openms/source/DATASTRUCTURES/Param.cpp_cppcheck_test (Failed)
        387 - src/openms/source/FORMAT/ConsensusXMLFile.cpp_cppcheck_test (Failed)
        726 - src/openms_gui/source/VISUAL/APPLICATIONS/TOPPViewBase.cpp_cppcheck_test (Failed)
        873 - src/topp/GNPSExport.cpp_cppcheck_test (Failed)
        905 - src/topp/OMSSAAdapter.cpp_cppcheck_test (Failed)
        1028 - src/openms/CMakeFiles/3.10.2/CompilerIdCXX/CMakeCXXCompilerId.cpp_cpplint_test (Failed)
        1029 - src/openms/OpenMS_autogen/mocs_compilation.cpp_cpplint_test (Failed)
        1654 - src/openms/source/SYSTEM/File.cpp_cpplint_test (Failed)
        1662 - src/openms/source/SYSTEM/UpdateCheck.cpp_cpplint_test (Failed)
        1719 - src/openms_gui/INIFileEditor_autogen/mocs_compilation.cpp_cpplint_test (Failed)
        1720 - src/openms_gui/OpenMS_GUI_autogen/mocs_compilation.cpp_cpplint_test (Failed)
        1721 - src/openms_gui/SwathWizard_autogen/mocs_compilation.cpp_cpplint_test (Failed)
        1722 - src/openms_gui/TOPPAS_autogen/mocs_compilation.cpp_cpplint_test (Failed)
        1723 - src/openms_gui/TOPPView_autogen/mocs_compilation.cpp_cpplint_test (Failed)
        1724 - src/openms_gui/qrc_resources.cpp_cpplint_test (Failed)
Errors while running CTest
Makefile:72: recipe for target 'test' failed
make: *** [test] Error 8
```

**Comment by oliveralka** [comment]
In my case only the ones below pop up:
macOS 10.15.7, OpenMS develop (a4496d9)
```
1 - potential_for_loop_copies (Failed)
1632 - src/openms/source/QC/TIC.cpp_cpplint_test (Failed)
1649 - src/openms/source/SYSTEM/File.cpp_cpplint_test (Failed)
1657 - src/openms/source/SYSTEM/UpdateCheck.cpp_cpplint_test (Failed)
```

**Comment by poshul** [comment]
Fixed in #5415
